syntax = "proto3";


message EnterGsInfo
{
  uint64 centre_node_id = 1;
}
